crypto/ec/ec2_smpl.c

```diff
@@ -956,8 +956,6 @@ const EC_METHOD *EC_GF2m_simple_method(void)
         ec_GF2m_simple_point_clear_finish,
         ec_GF2m_simple_point_copy,
         ec_GF2m_simple_point_set_to_infinity,
-        0, /* set_Jprojective_coordinates_GFp */
-        0, /* get_Jprojective_coordinates_GFp */
```

Comment:
> but I'd like to understand the reasoning.

Basically since EC support was integrated roughly two decades ago, that function pointer has been two-valued: `NULL` or `ec_GFp_simple_set_Jprojective_coordinates_GFp`. And most of the existing code that is still using it, is doing so out of laziness. Post scalar multiplication, the result should actually be immediately converted to affine to prevent multiple (costly) conversions upon serialization. Instead, that existing code is using the convenient fact that the coordinate system used in their internal `mul` functionality (i.e. `static` functions called from there) happens to use the same coordinate system (Jacobian Projective, even hardcoded in the fxn pointer name) as the stock `ecp_smpl` code. So just punting and magnifying the problem, so to speak.

So the fxn pointer just isn't getting much use. If you check a lot of the `EC_METHOD`s, it's mostly copy/paste boilerplate -- it's not really dynamic functionality. The original idea was sane -- 20 years ago, who knew what public facing interface we might need down the line as standards progress. But now we know better.

Reply:
Fair enough. Sounds reasonable.